### DISCUSSION TITLE
Where do I edit the config files in linux?

### DISCUSSION THREAD

**Discussion by denppa** [ahrm/sioyek#882] [Q&A]
Installed `sioyek` via flatpak on Fedora, then created a file at `~/.var/app/com.github.ahrm.sioyek/config/prefs_user.config`, but the configuration does not seem to reflect.
It just has the following line:
```
should_launch_new_window 1
```
Should I put the config somewhere else? The command prompt does not bring up the config file.

**Comment by denppa** [comment]
It seems like Gnome did not like to show the editor picker window on top of everything, so I missed it.

The config file at this time is stored at: `~/.var/app/com.github.ahrm.sioyek/config/sioyek/prefs_user.config/`

**Comment by atomdmac** [comment]
As of the latest Flatpak release, the `~/.var/app/com.github.ahrm.sioyek/config/sioyek/` did not exist for me so I had to manually create it and place my config files in there.  This works wonderfully (even updates the config without having to restart sioyek).  However, symlinks didn't work for me.  (I like to keep all of my config stuff in `~/.config` but this didn't work for me).
